scipy/special/_basic.py

```diff
@@ -3087,6 +3090,138 @@ def factorialk(n, k, exact=True):
     return _exact_factorialx_array(n, k=k)
 
 
+def stirling2(N, K, *, exact=True):
+    """Generate Stirling number(s) of the second kind.
+
+        Stirling numbers of the second kind count the number of ways to
+        partition a set with N elements into K non-empty subsets.
+
+        The values this function returns are calculated using a dynamic
```

Comment:
It's odd I didn't notice this, but the entire body of the docstring has an extra level of indentation.

Reply:
I didn't notice that either, thanks! (marking resolved)

Reply:
It gets easier to spot this by always having a linebreak after the initial `"""`.